src/components/accordion/types.ts

```diff
@@ -0,0 +1,24 @@
+import { ReactNode } from 'react'
+import { ExcludeNull } from 'types'
+import { type VariantProps } from 'class-variance-authority'
+import { accordionBaseVariant } from './accordion.classnames'
+
+export interface AccordionProps extends AccordionPropsExcludingNull {
+  id?: string
+  className?: string
+  title?: string
+  subtitle?: string
+  icon?: ReactNode
+  expanded?: boolean
+  divider?: AccordionDivider
+  contentClassname?: string
+  content?: () => ReactNode
```

Comment:
What is the difference between `content` and `customContent`  ?

Reply:
Needs to re-think the name here, designers are calling it `customContent` which means overriding the whole head of the `accordion` to a custom content of consumer's choice.